sklearn/linear_model/least_angle.py

```diff
@@ -978,8 +1018,9 @@ class LarsCV(Lars):
     def __init__(self, fit_intercept=True, verbose=False, max_iter=500,
                  normalize=True, precompute='auto', cv=None,
                  max_n_alphas=1000, n_jobs=1, eps=np.finfo(np.float).eps,
-                 copy_X=True):
+                 copy_X=True, positive=False):
```

Comment:
position of the positive param is not consistent for both classes. I would put it always as last param to avoid breaking code.